src/libs/layout/containers.ts

```diff
@@ -64,7 +67,7 @@ const getGapSize = (gap?: SpacingSize) => {
       return '8px';
     case SpacingSize.Medium:
       return '12px';
-    case SpacingSize.Big:
+    case SpacingSize.ExtraLarge:
```

Comment:
I think this should be Large in the current order, leave `ExtraLarge` for an additional bigger size